source/agora/node/main.d

```diff
@@ -61,7 +63,7 @@ private int main (string[] args)
             ex.message);
     }
 
-    Node node;
+    FullNode node;
```

Comment:
I would rename this to the `API` from `agora.api.FullNode`. Because technically it can be a Validator, and using the interface makes it more explicit that this is not necessarily always a `FullNode` object.

Reply:
This seems to require sufficient discussion.

Reply:
Currently, this the 66line refers to class `agora.node.FullNode`

Reply:
Ok it was just a suggestion, feel free to overrule. :)